hvplot/utilities.py

```diff
@@ -22,11 +22,14 @@ def show(obj, title=None, port=0, **kwargs):
         Allows specifying a specific port
     **kwargs: dict
         Additional keyword arguments passed to Panel show method.
+    Returns
+    -------
+    a Server | threading.Thread (if threaded=true)
```

Comment:
Are you sure it is returning a thread if `threaded=True`?

Reply:
If I'm following the code correctly, utility.show() is calling `[panel].pane.HoloViews(obj).show(title, port, **kwargs)` which then calls `serve` from panel.io.server.py.  Both of those have return values of `-> threading.Thread | Server:`
I've tested locally and it does work as expected there.

Reply:
Running this
``` python
import hvplot
import hvplot.pandas
import numpy as np
import pandas as pd

ts = pd.Series(np.random.randn(1000), index=pd.date_range("1/1/2000", periods=1000))
ts = ts.cumsum()
plot = ts.hvplot()
s = hvplot.show(plot, threaded=True)
print(type(s))
s.stop()
```

Outputs:
``` python
<class 'panel.io.server.StoppableThread'>
Launching server at http://localhost:35483
```

I think it should be `panel.io.server.StoppableThread` instead of `threading.Thread` to be correct.

(I also think the same is the case for Panel docstring, so if you want, you are welcome to open a PR on Panel)

Reply:
You're right it's `panel.io.server.StoppableThread | panel.io.server.Server`.  I'll update this PR and submit one for panel to fix its return value as well.